src/strategies/xdai-stake-lp/index.ts

```diff
@@ -0,0 +1,149 @@
+import { formatUnits } from '@ethersproject/units';
+import { getAddress } from '@ethersproject/address';
+import { multicall, subgraphRequest } from '../../utils';
+import examplesFile from './examples.json';
+
+export const author = 'maxaleks';
+export const version = '0.1.0';
+export const examples = examplesFile;
+
+const STAKE_TOKEN_ADDRESS = '0x0Ae055097C6d159879521C384F1D2123D1f195e6';
+const DECIMALS = 18;
+
+const exchanges = {
+  uniswap: {
+    lpTokenAddress: '0x3B3d4EeFDc603b232907a7f3d0Ed1Eea5C62b5f7',
+    subgraph: 'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2'
+  },
+  sushiswap: {
+    lpTokenAddress: '0x9Fc5b87b74B9BD239879491056752EB90188106D',
+    subgraph: 'https://api.thegraph.com/subgraphs/name/sushiswap/exchange'
+  }
+};
```

Comment:
Would be great to use these as default but can also overwrite addresses and URLs if passed from params (examples.json)